docs/process/roadmap.md

```diff
@@ -1,13 +1,67 @@
 # Roadmap
 
-Currently the team are focused on the 1.0 release, which should represent the
-set of features which enable users to replace the previous Classic Mac and
-Windows apps with this new Electron app.
+The following are our marketing releases as planned to date. They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time.
 
-Along with that, we'll be working on feedback and issues reported from the
-public beta. But feel free to open issues and start discussions on things
-you'd like to see in the app.
+## 1.0
```

Comment:
Maybe some words about what 1.0 represents, especially when compared to the Classic apps?